docs/docsource/operation/XDM_GUI/location_management/add_a_storage_location.rst

```diff
@@ -0,0 +1,272 @@
+**Bucket Match** box deactivates this feature: the prepending of bucket names is
+defeated, and the bucket structure in the host cloud is copied identically to
+the target cloud.
+
+.. important::
+
+   If the Bucket Match option is set, buckets in the target location cannot be
+   used as a CRR destination. |product| requires the bucket identifier in order to
+   manage the namespace for replication.
+
+Server-Side Encryption
+~~~~~~~~~~~~~~~~~~~~~~
+
+Encryption-based transfer protocols ensure your credentials and transmitted
+information are secure while in transit. The S3 API also offers encryption and
+key management services to protect information stored on cloud drives. From
+Orbit, clicking **Server Side Encryption** when setting up a location creates a
+location with encryption enabled for all objects stored there. Encryption is set
+at the bucket level, not at the object level. Object encryption is delegated to
+the cloud storage system.
+
+Server-side encryption is based on the x-amz-server-side-encryption
+header. Inquire with your cloud vendor to determine whether server-side
+encryption using x-amz-server-side-encryption is supported on their platform. A
+table is provided in this document, but vendors' offerings are subject to change
+without notice.
+
+If you have already created a bucket with server-side encryption enabled (SSE-S3
+protocol), clicking **Server Side Encryption** forces |product| to include
+``"x-amz-server-side-encryption": "AES256"`` in API calls to the cloud host (AWS
+or a vendor that supports the call). If valid credentials are provided, the cloud
+service provides the objects thus requested. 
+
+Target Helper Bucket for Multi-Part Uploads
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Google Cloud Storage solution imposes limitations on uploads that require
+specific workarounds. Among these is a 5 GB hard limit on uploads per command,
+which requires objects over this limit to be broken up, uploaded in parallel
+chunks, and on a successful upload reassembled in the cloud. |product| manages this
+complexity, in part, by using a "helper" bucket.
+
+.. note::
+
+   Google Cloud Storage also imposes a 1024-part cap on objects stored to its
+   locations (For all other backends, |product| caps the number of parts at
+   10,000). For data stored directly to GCP as the primary cloud, |product|
+   propagates this limitation forward to any other cloud storage services to
+   which Google data is replicated.
+
+Other Services: |product| Local, RING/sproxyd, and NFS
+------------------------------------------------------
+
+|product| Local Filesystem
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+|product| Local Filesystem has similar authentication requirements to AWS S3, but
+because it is a |product|-native filesystem, it shares authentication and related
+credentialing tasks, which are addressed elsewhere in the Orbit UI.
+
+For more information, see :ref:`|product| Local`.
+
+RING with sproxyd Connector
```

Comment:
I feel a link somewhere to RING or sproxyd might be helpful here, could be with a footer.

Reply:
Hold that thought. Pending RING-34432